quorum/scripts/generate_env_vars.sh

```diff
@@ -14,7 +14,8 @@ if [ "$1" = "" ] ; then
 fi
 
 GENESIS_JSON=`node "$pwd/quorum/scripts/generate_genesis.js" $@`
-STATIC_NODES=`node "$pwd/quorum/scripts/generate_static_nodes.js" $@`
+STATIC_NODES=`node "$pwd/quorum/scripts/generate_static_nodes.js" $1`
```

Comment:
Mocking the separation between the controller and the processor by making them have differente static-nodes files